packages/ds-medicare-gov/src/styles/_base.scss

```diff
@@ -115,3 +115,13 @@ svg {
   line-height: 20px;
   margin-bottom: $spacer-1;
 }
+
+/**
+ * Pagination overrides
+ */
+
+.ds-c-pagination {
+  .ds-c-button {
```

Comment:
NB: it looks like there's a class name "ds-c-pagination__nav" for the pagination buttons that could make this rule a a bit simpler

Reply:
It looks like that class name is only used for the chevron icons, and not the page numbers.